src/genesis/export_to_csv.py

```diff
@@ -0,0 +1,252 @@
+import requests
+import configparser
+import json
+import csv
+import os
+
+# Read the configuration file
+config = configparser.ConfigParser()
+config.read('config.ini')
+
+# Retrieve the API settings
+endpoint = config.get('api', 'endpoint')
+api_key = config.get('api', 'api_key')
+
+# Retrieve the output file path from the configuration
+output_file = config.get('files', 'output_file')
+output_csv = config.get('files','output_csv')
+
+# Retrieve options
+food_type = config.get('options', 'food_type')
+output_limit = int(config.get('options','limit',fallback=10000))
```

Comment:
There is a hard limit of 10K, it will fail if you try to call more. I'd do a check and throw an error with a message telling the user 10K is the max or set it to 10K and move forward e.g. 

`output_limit = min(int(config.get('options', 'limit', fallback=10000)), 10000)`